### PR TITLE
More GA.

### PR DESCRIPTION
**Pull request by AndreyNikitin**
https://codio.myjetbrains.com/youtrack/issue/codio-7195

### CODE REVIEW

File: source/partials/vendors/_google-analytics.erb [hunk 1]
```diff
@@ -3,6 +3,11 @@
 (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
 m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
 })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
-ga('create', '<%= codio_environment != 'production' ? 'UA-39233819-2' : 'UA-39233819-4' %>', 'auto');
+ga('create', '<%= codio_environment != 'production' ? 'UA-39233819-2' : 'UA-39233819-4' %>', 'auto', 'Everything');
```
Comment (MaximKraev) [inline source/partials/vendors/_google-analytics.erb]: `ga` of `f_ga`?

Comment (AndreyNikitin) [inline source/partials/vendors/_google-analytics.erb]: it is not javascript but template, so ga

**Comment by MaximKraev** [comment]
:+1:

File: source/partials/vendors/_google-analytics.erb (hunk 1 above)
Comment (MaximKraev) [inline source/partials/vendors/_google-analytics.erb]: thank you, get it

**Pull request merged**